lib/Transforms/LoopInvariantCodeMotion.cpp

```diff
@@ -128,6 +126,13 @@ void LoopInvariantCodeMotion::runOnOperation() {
   // the outer loop, which in turn can be further LICM'ed.
   getOperation()->walk([&](Operation *op) {
```

Comment:
We could also change this to:

walk([&](LoopLikeOpInterface loopLike) {
  ...
}